PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Managers/AssetPersistenceManager.m

```diff
@@ -144,7 +144,9 @@ - (void)pauseDownloadForEmbedCode:(NSString *)embedCode {
   // find a download in progress for the given embed code, pause the download.
   NSLog(@"[AssetPersistenceManager] Attempting to pause a download for embed code: %@", embedCode);
   
-  for (OOAssetDownloadManager *downloadManager in self.downloadsPendingAuth) {
+  NSMutableSet *downloadsPendingAuthCopy = self.downloadsPendingAuth.copy;
```

Comment:
since you're copying it, should it be a `NSSet` instead of `NSMutableSet`? Applies for all the places you did the change.

Reply:
I think we can create `NSArray` instead of `NSSet`. I created `NSMutableSet` for the same type of the original type, but I agree,`NSSet` is more clear. 

Changed to `NSSet`.